### PR TITLE
update guanmai config

### PR DESCRIPTION
**Pull request by BigBugaboo**
I need change this config, because data of search has some trouble, I had examine config in local.

### CODE REVIEW

File: configs/guanmai.json [hunk 1]
```diff
@@ -1,21 +1,17 @@
 {
   "index_name": "guanmai",
+  "js_render": true,
```
Comment (s-pace) [inline configs/guanmai.json]: Can you avoid to rely on client side rendering please?

Comment (BigBugaboo) [reply configs/guanmai.json]: sorry, can no.I have many document and so I use this way to develop for lesse time.if i use other way that  I will be weast more time.

Comment (BigBugaboo) [reply configs/guanmai.json]: Why is this not recommended this method?

Comment (BigBugaboo) [reply configs/guanmai.json]: if i do not add this,I can't get the exact path.

Comment (s-pace) [reply configs/guanmai.json]: Do you have a clear example please?
Did you run the crawl on your machine?

**Comment by BigBugaboo** [comment]
> I need change this config, because data of search has some trouble, I had examine config in local.

**Pull request closed without merging**